.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       fail-fast: false
       matrix:
         # Test against all versions supported by rubygems
-        ruby_version: ["2.7", "3.0", "3.1", "3.2", "3.3", "3.4.0-preview2"]
+        ruby_version: ["2.7", "3.0", "3.1", "3.2", "3.3", "3.4"]
```

Comment:
It seems https://github.com/oxidize-rb/rb-sys/pull/472/files already handles this, so I'll pull this commit.